source/manual/govuk-notify.html.md

```diff
@@ -26,15 +26,20 @@ for each environment):
   email updates about pieces of content on GOV.UK. It's our biggest sender of
   emails and regularly exceeds one million emails per day.
 
+  **Note:** currently (as of 30/06/2020) GOV.UK Notify has a maximum rate limit of 350
+  requests per second meaning we should adhere to this limit when using Notify to
+  deliver emails for our applications. An example of where we set this for
```

Comment:
```suggestion
  deliver emails for our applications. This is set in...
```

We only implement the limit in one place, since we have a single/unified method of sending email.